libsolidity/parsing/Parser.cpp

```diff
@@ -902,6 +918,21 @@ ASTPointer<EventDefinition> Parser::parseEventDefinition()
 	return nodeFactory.createNode<EventDefinition>(name, documentation, parameters, anonymous);
 }
 
+ASTPointer<ErrorDefinition> Parser::parseErrorDefinition()
+{
+	RecursionGuard recursionGuard(*this);
+	ASTNodeFactory nodeFactory(*this);
+	ASTPointer<StructuredDocumentation> documentation = parseStructuredDocumentation();
+
+	solAssert(*expectIdentifierToken() == "error", "");
+	ASTPointer<ASTString> name(expectIdentifierToken());
+
+	ASTPointer<ParameterList> parameters = parseParameterList({});
```

Comment:
You have just changed the grammar to have a custom parameter list definition for errors. Doesn't it also require parsing it in a special way here?

Reply:
No, the VarDeclParserOptions cover all these cases.